src/map/entities/baseentity.h

```diff
@@ -171,12 +171,12 @@ enum ENTITYFLAGS : uint16
 {
     FLAG_NONE           = 0x000,
     FLAG_INFO_ICON      = 0x001, // (I) Icon next to name
-
+    FLAG_SIZE_1         = 0x002, // Increases size - You can use this in combination with FLAG_SIZE_2 to the maximum size
```

Comment:
per the comment on line 176, please do not introduce these flags at this time. I've experiment with these a lot, and behavior that results is not universal for all all entities. We need to sort out our bytes in the mask in the project first, which don't align between entity types fully (this is in **base entity**, yet MOBs players and NPCs have  the same values do diff things do to being offset by a byte or 2)